CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Minio cookbook contribution guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+Minio cookbook is the collection of community generated detailed guides to set up Minio with other tools/platforms. If you have a 
+working setup that uses Minio, we welcome you to add your guide to the Cookbook. Please add the below setions in your guide before adding a 
```

Comment:
Thanks @GarimaKapoor good point. I updated the description accordingly. Please check